akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1460,12 +1460,41 @@ trait FlowOps[+Out, +Mat] {
    *
    * '''Cancels when''' downstream cancels
    *
-   * @param seed Provides the first state for extrapolation using the first unconsumed element
    * @param extrapolate Takes the current extrapolation state to produce an output element and the next extrapolation
    *                    state.
    */
   def expand[U](extrapolate: Out ⇒ Iterator[U]): Repr[U] = via(new Expand(extrapolate))
 
+  /**
+   * Allows a faster downstream to progress of a slower upstream.
```

Comment:
Incomplete sentence

Reply:
Gah, I have no idea how I missed that, sorry.